components/mixins/profile/profile-consent.example.1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "xdm:consentsAndPreferences": {
+    "xdm:consentOptions": {
+      "xdm:privacyPreferences": {
+        "xdm:dataCollection": {
+          "xdm:choice": "yes",
+          "xdm:timestamp": "2019-01-01T15:52:25+00:00",
+          "xdm:basisOfProcessing": "consent"
+        },
+        "xdm:deviceLinking": {
+          "xdm:basisOfProcessing": "vital_interest"
+        },
+        "xdm:anonymousAnalysis": {
+          "xdm:choice": "no"
+        }
+      },
+      "xdm:personalizationPreferences": {
+        "xdm:general": {
+          "xdm:choice": "yes",
+          "xdm:timestamp": "2019-01-01T15:52:25+00:00",
+          "xdm:basisOfProcessing": "consent"
+        }
+      },
+      "xdm:marketingPreferences": {
+        "xdm:preferredChannel": "email",
+        "xdm:general": {
+          "xdm:choice": "unknown"
+        },
+        "xdm:iotMessages": {
+          "xdm:choice": "yes",
+          "xdm:timestamp": "2019-01-01T15:52:25+00:00",
+          "xdm:basisOfProcessing": "legitimate_interest"
+        },
+        "xdm:pushNotifications": {
+          "xdm:choice": "yes",
+          "xdm:source": "OurApp"
+        },
+        "xdm:eMail": {
```

Comment:
is there a reason for using eMail vs email?

Reply:
@RussStringham let's use all lowecase "email" here to be consistent with other places like preferredChannel. This key is also important for CJM filtering

Reply:
this is a typo, should be xdm:email